app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,19 @@
+class AnswersController < ApplicationController
+
+  def create
+    @answer = Answer.new(answer_params)
+    @answer.user = User.last #current_user
```

Comment:
Why does User.last represent the current user?  I don't understand.  FYI, I put a helper method in the ApplicationController to return "current_user" as an object in any controller, but it will not be available until that branch is merged (I haven't done the tests for it yet).